.github/workflows/ci.yml

```diff
@@ -112,14 +112,20 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: dtolnay/rust-toolchain@stable
-      - uses: baptiste0928/cargo-install@v2
-        with:
-          crate: cargo-msrv
-          version: "^0.15.1"
+      - name: read msrv
+        id: msrv
+        run: |
+          msrv=$(grep -P "rust-version =" crates/pathfinder/Cargo.toml | awk '{print $3}' | tr -d '"')
+          echo Found msrc: $msrv
+          echo "MSRV=$msrv" >> "$GITHUB_OUTPUT"
+      - uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: ${{ steps.msrv.outputs.MSRV }}
+      - name: Install Protoc
+        uses: arduino/setup-protoc@v2
```

Comment:
We should either use this action everywhere or just keep using our `apt-get install` command line to install protoc. Is there any advantage of using this action?

Reply:
I just missed the others somehow, will change to apt-get